tracing-jersey/src/test/java/com/palantir/tracing/jersey/TraceEnrichingFilterTest.java

```diff
@@ -175,7 +175,10 @@ public void testTraceState_setsResponseStatus() {
         Span span = spanCaptor.getValue();
         assertThat(span.getOperation()).isEqualTo("Jersey: POST /trace");
         assertThat(span.getMetadata())
-                .containsEntry(TraceTags.HTTP_STATUS_CODE, Integer.toString(response.getStatus()));
+                .containsEntry(TraceTags.HTTP_STATUS_CODE, Integer.toString(response.getStatus()))
```

Comment:
nit: #containsEntry vs comparing the whole thing? I always prefer hasSameEntriesAs.

Reply:
I thought containsEntry chains were easiest to read, but I don't have much preference for these things. Should we update https://github.com/palantir/assertj-automation to replace one with the other?

Reply:
I meant more around one checks that you have some entries, and another checks that you have some entries AND NOTHING ELSE.

Reply:
Ah I see, it was my goal to avoid breaking the test upon additive updates, but we could do an exhaustive test instead.